simple-meeting/.gitignore

```diff
@@ -1 +1,2 @@
-node_modules/
\ No newline at end of file
+node_modules/
+simple-meeting\node_modules
```

Comment:
huh?

Reply:
line 1 is probably incorrect, since its `/node_modules` but the things are installed under the folders of `simple-meeting\node_modules`

Reply:
this file is `simple-meeting/.gitignore` then the folder should be relative to `simple-meeting`  - am i missing something?

Reply:
overall, the organization of the repo is messy, but

1. under which folder did you run `npm i` ? 
2. should `.gitignore` be at the root?

Reply:
Ok yeah I see what you mean.

I think the `.gitignore` should be at the root, since people clone the entire `iotum-samples` repo.